src/main/java/org/LogAggregatorTool/writer/LogFilesWriter.java

```diff
@@ -0,0 +1,27 @@
+package org.LogAggregatorTool.writer;
+
+import org.LogAggregatorTool.utility.Constants;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class LogFilesWriter {
+    public void writeToOutputFile(ArrayList<String> sortedTimestampsList, String outputFilePath, HashMap<String, ArrayList<String>> timestampToLogStatementMap) throws IOException {
+        BufferedWriter outputFile = new BufferedWriter(new FileWriter(outputFilePath));
+        for (String timestamp : sortedTimestampsList) {
```

Comment:
Any valid reason to use sortedTimestampsList separately ?
When Map can hold both timeStamp and data 
Try to use existing functionalities